### PR TITLE
:bug: Feat/import from json

### PR DESCRIPTION
**Pull request by guillaumeprevost76**

### CODE REVIEW

File: saagieapi/env_vars/env_vars.py [hunk 1]
```diff
@@ -415,3 +416,74 @@ def export(self, project_id, output_folder: str, error_folder: Optional[str] = "
...
+        try:
+            with open(json_file, "r") as file:
+                env_var_info = json.load(file)
+        except Exception as e:
+            logging.warning("Cannot open the JSON file %s", json_file)
+            logging.error("Something went wrong %s", e)
+            return False
+
+        try:
+            env_var_id = env_var_info["id"]
```
Comment (Qiwei08) [inline saagieapi/env_vars/env_vars.py:444]: env_var_id not used

File: saagieapi/env_vars/env_vars.py [hunk 2]
```diff
@@ -415,3 +416,74 @@ def export(self, project_id, output_folder: str, error_folder: Optional[str] = "
...
+            env_var_name = env_var_info["name"]
+            env_var_scope = env_var_info["scope"]
+            env_var_value = env_var_info["value"]
+            env_var_description = env_var_info["description"]
+            env_var_is_password = env_var_info["isPassword"]
+
```
Comment (Qiwei08) [inline saagieapi/env_vars/env_vars.py:450]: Should we add a test to see if the value of the variable is null ? 

If the value of the variable is null , this function will return the following error: 
`TransportQueryError: {'message': 'Environment variable is invalid', 'extensions': {'value': 
"Environment variable value can't be null", 'classification': 'ValidationError'}}`

File: tests/integration/resources/import/pipeline_non_existing_jobs.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
+{
```
Comment (Qiwei08) [inline tests/integration/resources/import/pipeline_non_existing_jobs.json:1]: Can we store this file in `tests/integration/resources/import/pipeline/pipeline_non_existing_jobs.json` ?

File: tests/integration/resources/import/pipeline_existing_jobs.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
+{
```
Comment (Qiwei08) [inline tests/integration/resources/import/pipeline_existing_jobs.json:1]: Can we store this file in `tests/integration/resources/import/pipeline/pipeline_existing_jobs.json` ?

File: tests/integration/resources/import/pipeline_non_existing_jobs.json (hunk 3 above)
Comment (guillaumeprevost76) [reply tests/integration/resources/import/pipeline_non_existing_jobs.json:1]: You're right, I will change that

File: tests/integration/resources/import/pipeline_existing_jobs.json (hunk 4 above)
Comment (guillaumeprevost76) [reply tests/integration/resources/import/pipeline_existing_jobs.json:1]: You're right, I will change that

File: saagieapi/env_vars/env_vars.py (hunk 1 above)
Comment (guillaumeprevost76) [reply saagieapi/env_vars/env_vars.py:444]: unused variable deleted.
I also checked import_from_json on job and pipeline. I found an unused variable on job and deleted it.

File: saagieapi/env_vars/env_vars.py (hunk 2 above)
Comment (guillaumeprevost76) [reply saagieapi/env_vars/env_vars.py:450]: Sure, I will add a test to put an empty value in the variable "env_var_value" if the value found in the JSON file is null or not found.

**Review by Qiwei08 — APPROVED** [review]
